test/unit/mongo_logger.test.ts

```diff
@@ -683,20 +677,13 @@ describe('class MongoLogger', function () {
               logDestination: stream
             } as any);
 
-            const EMERGENCY = 0;
             // Calls all severity logging methods with a level less than or equal to what severityLevel
-            for (
-              let l = SEVERITY_LEVEL_MAP.getNumericSeverityLevel(severityLevel);
-              l >= EMERGENCY;
-              l--
-            ) {
-              const severity = SEVERITY_LEVEL_MAP.getSeverityLevelName(l);
+            for (let i = index; i >= 0; i--) {
+              const severity = severities[i];
```

Comment:
if we type `severities` as `SeverityLevel[]`, we should be able to remove the cast in line 683